lib/minify.js

```diff
@@ -155,8 +159,8 @@ var minify = function(files, options) {
     }
 
     var mappingUrlPrefix = "\n//# sourceMappingURL=";
-    if (options.sourceMapInline) {
-        stream += mappingUrlPrefix + "data:application/json;charset=utf-8;base64," + new Buffer(source_map).toString("base64");
+    if (options.sourceMapInline && exports.base64Decoder !== defaultBase64Decoder) {
+        stream += mappingUrlPrefix + "data:application/json;charset=utf-8;base64," + exports.base64Decoder(source_map);
```

Comment:
`&& exports.base64Decoder !== defaultBase64Decoder` should be dropped, as the function is defined in all cases and you want it to throw.